Bottom: cbce57abe0452caece2304b0f5233fd7342500a7
Top:    30cbe7dd3e8d35d6a069418fdce9e9d86126d92c
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 20:31:08 +0000

Disable small edge EPSILON hack for computing edge normals


---

diff --git a/src/hid/step/object3d_step.c b/src/hid/step/object3d_step.c
index a04d8eb..468d966 100644
--- a/src/hid/step/object3d_step.c
+++ b/src/hid/step/object3d_step.c
@@ -227,7 +227,7 @@ object3d_to_step_body_fragment (step_file *step,
           dir_y = dv->y - ov->y;
           dir_z = dv->z - ov->z;
 
-#if 1
+#if 0
           /* XXX: This avoids the test file step_outline_test.pcb failing to display properly in freecad when coordinates are slightly rounded */
           if (dir_x < EPSILON && -dir_x < EPSILON &&
               dir_y < EPSILON && -dir_y < EPSILON &&
